### PR TITLE
Implement "remove role"

### PR DESCRIPTION
**Pull request by gjvis**
~~This has a placeholder design in place~~

Be sure to remove roles both with and without Javascript enabled.

### CODE REVIEW

**Comment by gjvis** [comment]
ok, this is now ready to go I think – its viewable here: https://work-you-could-do-pr-76.herokuapp.com

@nathancashmoredwp, @jessstubenbord can you review please?

**Comment by jessstubenbord** [comment]
looks good

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -45,4 +44,11 @@
   </main>
 <% end %>
 
+<% content_for :body_end do %>
+  <%= javascript_include_tag "application" %>
+  <% if Rails.env.test? %>
```
Comment (nathancashmoredwp) [inline app/views/layouts/application.html.erb]: Does this mean when the tests run they run against the non js version ?

**Comment by nathancashmoredwp** [comment]
:bikini... make it so and i'll point to master or at least create another fork for the demo.

**Pull request merged**